### PR TITLE
[18.0][MIG] base_time _window: Migration to 18.0

### PR DESCRIPTION
**Pull request by Kimkhoi3010**
It found only 1 commit to port:
It was ignored, i added them to blacklist: [4e65739](https://github.com/Kimkhoi3010/server-tools/pull/4/commits/4e6573981cebb79503733c46356af7692e5ddc4c)

### CODE REVIEW

**Review by nguyenminhchien — APPROVED** [review]

File: base_time_window/readme/CREDITS.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-The development of this module has been financially supported by:
+The migration of this module from 17.0 to 18.0 was financially supported by:
```
Comment (nilshamerlinck) [inline base_time_window/readme/CREDITS.md]: why not both? please don't erase the past

File: .eslintrc.yml
Comment (nilshamerlinck) [inline .eslintrc.yml:1]: probably need to rebase your 18.0 branch

**Review by nilshamerlinck — CHANGES_REQUESTED** [review]
oca PR is red

**Comment by Kimkhoi3010** [comment]
Sorry @nilshamerlinck, @nguyenminhchien, this PR close by my mistake.
This PR [#5](https://github.com/Kimkhoi3010/server-tools/pull/5) takes over

**Pull request closed without merging**